src/polkaswap-connect-wallet.md

```diff
@@ -0,0 +1,20 @@
+# Connect Wallet
+
+1.  Go to [Polkaswap.io](https://polkaswap.io) and click on the "Connect account" button at the top of the page.\
+
+![](.gitbook/assets/connect-wallet.png)
+
+2.  You will see a list of supported wallet options to connect, including [Fearless Wallet](https://fearlesswallet.io), Polkadot.js, Subwallet, and Talisman.\
+
+![](.gitbook/assets/choose-wallet-to-connect.png)
+
+3.  Select the wallet extension installed on your browser. For example, if you are using Fearless Wallet, click on the Fearless Wallet option
+4.  A pop-up window will appear from your selected wallet extension, asking you to confirm the connection to Polkaswap. Click "Allow" or "Confirm" to proceed.
```

Comment:
```suggestion
4. A pop-up window will appear from your selected wallet extension, asking you to confirm the connection to Polkaswap. Click "Allow" or "Confirm" to proceed.
```
why is it `or` in `"Allow" or "Confirm"`?

Reply:
No idea, fixed now